components/CodeViewer.vue

```diff
@@ -175,31 +198,36 @@ export default class extends Vue {
 
   public removeEditor(): void {
     var item = this.monacoId && document.getElementById(this.monacoId);
-    while (item && item.hasChildNodes()) {   
+    while (item && item.hasChildNodes()) {
       item.firstChild && item.removeChild(item.firstChild);
     }
   }
 
   public reloadEditor(): void {
-    this.editor = this.monacoId && (<any>window).monaco.editor.create(document.getElementById(this.monacoId), {
-            language: this.codeType,
-            value: this.data$,
-            lineNumbers: "on",
-            fixedOverflowWidgets: true,
-            wordWrap: "on",
-            scrollBeyondLastLine: false,
-            wordWrapMinified: true,
-            wrappingIndent: "indent",
-            fontSize: 16,
-            minimap: { enabled: false },
-            onCodeChange: this.onCodeChange,
-            onDidChangeModelDecorations: this.onDecorationsChange,
-            editorDidMount: this.editorMount
-        });
+    this.editor =
```

Comment:
mention for monaco editor issue
https://github.com/microsoft/monaco-editor/issues/2050